packages/individual-packages/README.md

```diff
@@ -0,0 +1,3 @@
+In this directory we generate at build time all individual packages in V2
+that we release separately from `aws-cdk-lib`
+(right now, those are the unstable packages).
```

Comment:
Yes for other packages, not `cx-api` or `assert` though - the L3s that are shipped today as part of `aws-cdk-lib` (and shouldn't be, in my opinion).

Reply:
I would suggest using the same mechanisms for all packages, rather than have two different ways of doing this. 

consistency reduces cognitive overhead in understanding our repo

Reply:
Done.